packages/x-date-pickers/src/DateField/useDateField.ts

```diff
@@ -47,9 +47,10 @@ const useDefaultizedDateField = <TDate, AdditionalProps extends {}>(
   const defaultDates = useDefaultDates<TDate>();
 
   return {
-    disablePast: false,
-    disableFuture: false,
     ...props,
+    disablePast: props.disablePast ?? false,
```

Comment:
The default value was not applied when the hook received `disablePast: undefined` (same for the others).
Which for some fields is the case depending on the component implementation.

Reply:
Should we add test in validation to check the default values are applied?

Reply:
Could it be done in your `describeValidation` ?